jena/TemporalIndexProcessor/test/com/bbn/parliament/jena/graph/index/temporal/query/QueryEdgeCaseTest.java

```diff
@@ -181,7 +177,11 @@ private static void printQuerySolution(QuerySolution querySolution) {
 	}
 
 	private static void checkResults(ResultSet rs, String... results) {
-		List<String> values = Arrays.asList(results);
+		Set<String> values = new HashSet<>();
+		for (String result : results) {
+			values.add(result);
+		}
+
```

Comment:
An alternate, one-line formulation:  `Set<String> values = Arrays.stream(results).collect(Collectors.toSet());`

Reply:
I am trying to ensure that the logic further down that removes items from the `Set` using `Iterator::remove()` will work.  The `Set` implementation used by `Collectors::toSet()`, but I am not certain.  I know that `HashSet` supports this functionality.